### DISCUSSION TITLE
Is Arachne gone??

### DISCUSSION THREAD

**Discussion by bangor-flyer** [supermerill/SuperSlicer#3845] [General]
Hey Guys sorry in advance if I have the blinkers on..
I was convinced I was using Arachne in SS but when going to look for the setting now I can't find it?
Am I doing something stupid or has it been removed?
Many thanks in Advance guys 
take care

**Comment by bangor-flyer** [comment]
Found answer

**Reply by digitalentropy** [reply]
What....was your answer? This is a very unhelpful comment.

**Reply by bangor-flyer** [reply]
As in I was to quick to ask the question and not look hard enough for my answer and the fact that the guys have already answered it in previous posts
